web/satellite/vuetify-poc/src/components/TeamTableComponent.vue

```diff
@@ -2,40 +2,38 @@
 // See LICENSE for copying information.
 
 <template>
-    <v-card min-height="24" :border="0" class="mb-2 elevation-0">
-        <v-row v-if="selectedMembers.length > 0" class="justify-end align-center ma-0">
+    <v-row v-if="selectedMembers.length > 0" class="justify-end align-center ma-0">
+        <v-col>
             <p>{{ selectedMembers.length }} user{{ selectedMembers.length !== 1 ? 's' : '' }} selected</p>
-        </v-row>
-    </v-card>
+        </v-col>
+    </v-row>
     <v-card variant="flat" :border="true" rounded="xlg">
-        <v-row align="center" class="ma-0">
-            <v-col v-if="selectedMembers.length" class="px-0 mr-0 ml-2 mt-2" cols="auto">
-                <v-btn
-                    class=" text-caption"
-                    prepend-icon="mdi-trash-can-outline"
-                    variant="outlined"
-                    color="default"
-                    @click="showDeleteDialog"
-                >
-                    Remove
-                </v-btn>
-            </v-col>
-
-            <v-col class="px-0 mx-2 mt-2">
-                <v-text-field
-                    v-model="search"
-                    label="Search"
-                    prepend-inner-icon="mdi-magnify"
-                    single-line
-                    variant="solo-filled"
-                    flat
-                    hide-details
-                    clearable
-                    density="comfortable"
-                    rounded="lg"
-                />
-            </v-col>
-        </v-row>
+        <div v-if="selectedMembers.length" cols="auto">
```

Comment:
This button is positioned above the search field instead of beside it.

Reply:
Thank you Jeremy, i missed this one. Fixed now.